pkg/utils/struct_test.go

```
@@ -13,7 +13,7 @@ type testStruct struct {
 	Age     int    `json:"age" xml:"age"`
 	Email   string `json:"email"`
 	Active  bool   `json:"active"`
-	private string `json:"private"` //nolint:govet,staticcheck // unexported field used for testing
+	private string // field without tags for testing unexported
```

golangci-lint (2.12.2)

[error] field private is unused

(unused)